python/sdk/setup.py

```diff
@@ -25,7 +25,7 @@
     "boto3>=1.9.84",
     "caraml-upi-protos>=0.3.1",
     "certifi>=2017.4.17",
-    "Click>=7.0",
+    "Click>=7.0,<8.1.4",
```

Comment:
Newer [versions](https://click.palletsprojects.com/en/8.1.x/changes/#changes) of this library has a breaking change causing errors in the CI. Eg: https://github.com/caraml-dev/merlin/actions/runs/5567016741/jobs/10168580897

So, pinning the version to <8.1.4